PowerShell-PoC-Transcript-2016-10-21-0834.log

```diff
@@ -0,0 +1,1065 @@
+**********************
```

Comment:
Please, remove the transcript file from the commit, we don't want that uploaded into the solution